macro/build_geometry.C

```diff
@@ -127,16 +128,21 @@ void build_geometry(FairRunSim* run = nullptr)
   // beam pipe
   if (isActivated("PIPE")) {
 #ifdef ENABLE_UPGRADES
-    if (isActivated("IT3") || isActivated("TRK")) {
-      run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe", 1.6f, 0.05));
-    } else {
-      run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe"));
+    if (isActivated("IT3")) {
+      run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe", 1.6f, 0.05f));
```

Comment:
I think you are missing the `else` branch here to allow for the simulation of the standard setup with `ENABLE_UPGRADES` active.

Reply:
Isn't it the default behaviour?
It enables the changed beampipe only if `-m PIPE ITS3` is required, the Run3 stadnard one otherwise.

Reply:
True, I always manage to mess with this.

Reply:
Maybe I am missing something, but I don't think this is correct yet. The standard beam pipe should be used if `ENABLE_UPGRADES` is not set or the standard ITS is used. At the moment it would always be added. I think the code should read something like this:
```suggestion
#ifdef ENABLE_UPGRADES
     if (isActivated("IT3")) {
       run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe", 1.6f, 0.05f));      
     } else { 
       run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe"));
     }
#else
     run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe"));
#endif
```
Does this make sense to you?

Reply:
It does, I updated the PR, thx.